azext_iot/tests/iothub/core/test_iothub_storage_int.py

```diff
@@ -23,7 +23,7 @@
 STORAGE_ROLE = "Storage Blob Data Contributor"
 CWD = os.path.dirname(os.path.abspath(__file__))
 user_managed_identity_name = generate_generic_id()
-USER_IDENTITY_SETUP_MAX_ATTEMPTS = 5
+SETUP_MAX_ATTEMPTS = 3
 
 
 class TestIoTStorage(IoTLiveScenarioTest):
```

Comment:
Can you please post a pipeline run.

Reply:
all the ones in the pr description are runs with this - will be posting more to get a feel for how often the storage tests fail now